### PR TITLE
Added ignoring of emulator shortcuts to keyboard_pad_handler

### PR DESCRIPTION
**Pull request by PeterMcteague**
This fixes an issue where pressing keyboard shortcuts will input them to
the game.

This should fix issue #3280.

I've  tested on Windows and have found it to work for at least full-screening, didn't test the others but they are handled. I think the solution could be improved by us having something which checks for these as similar code also crops up in gs_frame , so there's a little bit of redundancy here and you'd have to update this every time a new keyboard shortcut is added.

### CODE REVIEW

File: rpcs3/keyboard_pad_handler.cpp [hunk 1]
```diff
@@ -60,9 +60,40 @@ void keyboard_pad_handler::keyPressEvent(QKeyEvent* event)
...
-	event->ignore();
+	switch (event->key())
+	{
+		case Qt::Key_L:
+			if (!(event->modifiers() == Qt::AltModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		case Qt::Key_Return:
+			if (!(event->modifiers() == Qt::AltModifier)) { Key(event->key(), 1); }
+			break;
```
Comment (Megamouse) [inline rpcs3/keyboard_pad_handler.cpp]: Switched lines

**Comment by Megamouse** [comment]
Cant test right now @flash-fire

**Comment by AniLeo** [comment]
Please fix switched lines (you're calling event->ignore() after breaking) so we can merge

**Comment by AniLeo** [comment]
Travis broke, not an issue from the Pull Request itself. We're looking into it.

File: rpcs3/keyboard_pad_handler.cpp [hunk 2]
```diff
@@ -60,9 +60,40 @@ void keyboard_pad_handler::keyPressEvent(QKeyEvent* event)
...
+			if (!(event->modifiers() == Qt::ControlModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		case Qt::Key_E:
+			if (!(event->modifiers() == Qt::ControlModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		default:
+			Key(event->key(), 1);
+			event->ignore();
```
Comment (Megamouse) [inline rpcs3/keyboard_pad_handler.cpp]: I didn't really look into it yet but shouldn't default just break?

**Comment by Megamouse** [comment]
maybe we should implement a shortcut collection somewhere instead of coding them everytime

File: rpcs3/keyboard_pad_handler.cpp (hunk 2 above)
Comment (PeterMcteague) [reply rpcs3/keyboard_pad_handler.cpp]: I just went with the same behaviour that was there before. I can check if break would work.

Comment (Megamouse) [reply rpcs3/keyboard_pad_handler.cpp]: oh sorry, I somehow missed the old code on my phone. nvm then

**Pull request merged**